doc/release-notes/9.0.md

```diff
@@ -0,0 +1,263 @@
+# Release notes for Gluster 9.0
+
+This is a major release that includes a range of features, code improvements and stability fixes as noted below.
+
+A selection of the key features and changes are documented in this page.
+A full list of bugs that have been addressed is included further below.
+
+- [Announcements](#announcements)
+- [Major changes and features](#major-changes-and-features)
+- [Major issues](#major-issues)
+- [Bugs addressed in the release](#bugs-addressed)
+
+## Announcements
+
+1. Releases that receive maintenance updates post release 9 is 8 
+([reference](https://www.gluster.org/release-schedule/))
+
+2. Release 9 will receive maintenance updates around the 30th of every month
+for the first 3 months post release (i.e Jan'21, Feb'21, Mar'21). Post the
```

Comment:
just confused as at line 18, it mentions 30th of every month

Reply:
Its the month and Year....line 18 is the date.